### PR TITLE
BLO-530: Implement Lockscreen redesign

### PR DESCRIPTION
**Pull request by bluecco**

### CODE REVIEW

File: packages/extension/src/ui/features/lock/LockScreen.tsx [hunk 1]
```diff
@@ -1,97 +1,98 @@
...
+            <>
+              <StickyGroup>
+                <Button
+                  gap="2"
+                  colorScheme="primary"
+                  type="submit"
+                  disabled={!isDirty || isSubmitting}
+                  width="100%"
+                >
+                  {isLoading && (
```
Comment (janek26) [inline packages/extension/src/ui/features/lock/LockScreen.tsx:83]: chakra button component ships with buildin `isLoading` prop! :) 
https://chakra-ui.com/docs/components/button/props

Comment (bluecco) [reply packages/extension/src/ui/features/lock/LockScreen.tsx:83]: nice, will update the component

Comment (bluecco) [reply packages/extension/src/ui/features/lock/LockScreen.tsx]: updated @janek26

**Review by janek26 — APPROVED** [review]
works like a charm!

**Pull request merged**